backend/src/main/java/org/latexscribe/LatexScribe/service/impl/DocumentServiceImpl.java

```diff
@@ -61,5 +61,8 @@ public List<Document> findByTag(DocumentTag tag) {
         return documentRepository.findByTag(tag);
     }
 
-
+    @Override
+    public List<Document> findByName(String name) {
+        return documentRepository.findByNameContainsIgnoreCase(name);
+    }
```

Comment:
It should find the documents for the specific user that is logged in, otherwise it will get everyones documents.

Reply:
cant test it
i get 403 een when im logged in

Reply:
When you login the server gives you a an access token, you need to used that access token by putting it in the `Authorization` field in the request.

See: https://www.tutorialspoint.com/postman/postman_authorization.htm

Reply:
I will add on more endpoint for listing all documents for the user

Reply:
Sure, if you are going to add one, then put it at `GET /api/v1/documents`, just like we do with the searching in this PR we can get all the documents associated with user.